package.json

```diff
@@ -23,7 +24,7 @@
     "prop-types": "^15.5.10",
     "querystring": "^0.2.0",
     "react": "16.0.0-alpha.12",
-    "react-instantsearch": "^4.0.10",
+    "react-instantsearch": "^4.1.0",
```

Comment:
We may need to downgrade this as well, if we move back to 0.46

Reply:
I see. I will check and updated it accordingly.